.github/workflows/Release.yml

```diff
@@ -37,90 +37,25 @@ jobs:
           name: DLT-Mac
           path: DLT-macOS.zip
 
-  buildLinux:
-    name: Build ${{ matrix.ubuntu }}
-    strategy:
-      matrix:
-        ubuntu: [ ubuntu-20.04 ]
-    runs-on: ${{ matrix.ubuntu }}
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-          submodules: true
-      - name: install build environment
-        run: scripts/linux/install.sh
-      - name: Build project
-        run: scripts/linux/build.sh
-      - name: Make artifact executable
-        run: chmod -R +x build/dist
-      - name: Archive artifact
-        run: zip DLT-Linux.zip -r build/dist
-      - name: Upload DLT artifact
-        uses: actions/upload-artifact@v4
-        with:
-          name: DLT-Linux
-          path: DLT-Linux.zip
 
-  buildWindows:
-    name: Build Windows
-    strategy:
-      matrix:
-        task: [ parser, sdk ]
-    runs-on: windows-2019
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-          submodules: true
-      - name: Install qt
-        uses: jurplel/install-qt-action@v4
-        with:
-          mirror: http://ftp.fau.de/qtproject/
-          version: 5.15.2
-      - name: Build ${{ matrix.task }}
-        run: >
-          $env:WORKSPACE=$env:GITHUB_WORKSPACE; $env:QTDIR=$env:Qt5_DIR.Replace('/','\');
-          ./build_${{ matrix.task }}_windows_qt5_MSVC.bat
-      - name: Archive artifact
-        run: >
-          Compress-Archive -Path .\build\dist -DestinationPath
-          DLT-Windows-${{ matrix.task }}.zip
-      - name: Upload DLT artifact
-        uses: actions/upload-artifact@v4
-        with:
-          name: DLT-Windows-${{ matrix.task }}
-          path: DLT-*.zip
+
 
   createRelease:
     name: Create release
     runs-on: ubuntu-20.04
-    needs: [ buildMac, buildLinux, buildWindows ]
```

Comment:
I guess, you did this just for development purpose, right ?

Reply:
Yes was done to make development faster. re enabled windows and ubuntu